tests/test_terra_notebook_utils.py

```diff
@@ -116,6 +118,30 @@ def test_copy(self):
             key = f"gs://{WORKSPACE_BUCKET}/test_oneshot_object_{uuid4()}"
             drs.copy(self.drs_url, key)
 
+    @testmode("controlled_access")
+    def test_copy_batch(self):
+        drs_urls = {
+            "NWD522743.b38.irc.v1.cram.crai": "drs://dg.4503/95cc4ae1-dee7-4266-8b97-77cf46d83d35",  # 1631686 bytes
+            # "NWD961306.freeze5.v1.vcf.gz": "drs://dg.4503/6e73a376-f7fd-47ed-ac99-0567bb5a5993",  # 2679331445 bytes
```

Comment:
Temporarily inaccessible?  XD  The size is a good comment.

Reply:
Including the 2GB files increases the duration of the test, but they're still useful for spot testing large files now and then.